### ISSUE TITLE
Wrong glyphs on default install

### ISSUE THREAD

**Issue by GitClickOk** [JanDeDobbeleer/oh-my-posh2#294]
I installed on-my-posh using the instructions from readme.md. My profile file is like this:

```
Import-Module posh-git
Import-Module oh-my-posh
Set-Prompt
Set-Theme Paradox
```
But my output appear like this:
![Capture 1838](https://user-images.githubusercontent.com/19438005/91242241-47081000-e71d-11ea-88c6-cf3786736292.png)

Do I need install some custom font? I thought the default install will show at least these bottom and top of arrows, like in the screenshots:
![Capture 1839](https://user-images.githubusercontent.com/19438005/91242332-87678e00-e71d-11ea-86f3-bdf72a2fea9b.png)

Am I doing something wrong?

(If it helps, I'm using Windows Terminal, but the problem happens in any other terminal)

**Comment by nwykes** [comment]
Please read the Readme.  It's covered in the prerequisites.  You need to install font with the glyphs.

**Comment by GitClickOk** [comment]
@nwykes 
Sorry, I'm not an expert in Windows Terminal, Powershell/PSCore or oh-my-posh. Probably this is pretty obvious for some, but not for me.

If it matters, I have installed this font:
https://github.com/ryanoasis/nerd-fonts/blob/master/patched-fonts/Meslo/M/Regular/complete/Meslo%20LG%20M%20Regular%20Nerd%20Font%20Complete%20Windows%20Compatible.ttf

The font name looks with the name cited in readme.md (_Meslo LG M Regular for Powerline Nerd Font_). I have downloaded, right clicked and installed, then closed the terminal, reopened and nothing happened. I checked the Windows Fonts app, and this font  is installed.

So I'm sorry, I really need some kind of "ELI5" for this problem.

**Comment by nwykes** [comment]
You would need to tell the terminal to use the font.  installing the font into windows doesn't change the font the terminal is using.  Windows Terminal has a drop down with "Settings" which will open the settings.json in the default editor.  Documentation for the settings can be found https://docs.microsoft.com/en-us/windows/terminal/customize-settings/profile-settings#text-settings

**Comment by GitClickOk** [comment]
@nwykes 
Thanks, I did it work now. For reference, I did like you said (Terminal Dropdown -> Settings). I have added this lines:

```
    "profiles":
    {
        "defaults":
        {
            "fontFace" : "MesloLGM NF",
            "fontSize" : 10
        }
     }
```
